### PR TITLE
Feature/color code reciprocal traces

### PR DESCRIPTION
**Pull request by timjarsky**
Colors traces from reciprocal connections red if the argument "--Recip" is passed.

Removes mismatched event types e.g. a negative going PSP from an excitatory Cre-type

### CODE REVIEW

File: synapse_comparison.py [hunk 1]
```diff
@@ -84,13 +88,29 @@ def first_pulse_plot(expt_list, name=None):
...
                     avg_amp.t0 = 0
                     avg_ests.append(avg_est)
                     base = float_mode(avg_amp.data[:int(10e-3 / avg_amp.dt)])
                     amp_base_subtract.append(avg_amp.copy(data=avg_amp.data - base))
-                    amp_plots.plot(avg_amp.time_values, avg_amp.data - base)
+
+                    current_connection_HS = post, pre
+                    if len(expt.connections) > 1 and args.Recip is True:
+                        for x in expt.connections:
+                            if cmp(current_connection_HS, x) == 0:  # determine if a reciprocal connection
```
Comment (campagnola) [inline synapse_comparison.py]: Does `if x == current_connection_HS` work here?

File: synapse_comparison.py [hunk 2]
```diff
@@ -84,13 +88,29 @@ def first_pulse_plot(expt_list, name=None):
...
+                                amp_plots.plot(avg_amp.time_values, avg_amp.data - base, pen={'color': 'r', 'width': 1})
+                                break
+                            else:  # reciprocal connection was not found
+                                 amp_plots.plot(avg_amp.time_values, avg_amp.data - base)
```
Comment (campagnola) [inline synapse_comparison.py]: Seems like this will cause individual synapses to be plotted multiple times.

File: synapse_comparison.py [hunk 3]
```diff
@@ -31,6 +32,9 @@
                     help='cre-type must also be specified')
 parser.add_argument('--age', type=str, help='Enter age ranges separated by ",". Ex 40-50,60-70.'
                     '' 'cre-type must also be specified')
+# Optional argument
+parser.add_argument('--Recip', action='store_true', default=False, dest='Recip',
```
Comment (campagnola) [inline synapse_comparison.py]: Flags are lowercase by convention (because lazy fingers)

File: synapse_comparison.py (hunk 1 above)
Comment (timjarsky) [inline synapse_comparison.py]: @campagnola It does.

File: synapse_comparison.py (hunk 3 above)
Comment (timjarsky) [inline synapse_comparison.py]: @campagnola OK

File: synapse_comparison.py (hunk 2 above)
Comment (timjarsky) [inline synapse_comparison.py]: @campagnola I think I've fixed this.

**Comment by campagnola** [comment]
@scseeman can you test this?

**Comment by campagnola** [comment]
Thanks!

**Pull request merged**